config.yaml

```diff
@@ -9,8 +27,7 @@ modules:
   load:
       - access-esm1p5/2024.05.0
```

Comment:
@jo-basevi has just deployed a version of `payu` that supports the `module use/load` features. When this version of `payu` is used it throws an error that there were multiple versions of models available because `module use /g/data/vk83/modules` also picks up the modules in the `access-models` subdirectory.

The solution is to change to
```yaml
modules:
  use:
      - /g/data/vk83/modules/
  load:
      - access-esm1p5/2024.05.0
```
which should be a no-op if `/g/data/vk83/modules/` is already `use`'d (I just tested outside of `payu` and it works fine, but please do check).

Reply:
Just added this in and running the simulation with these changes is working!